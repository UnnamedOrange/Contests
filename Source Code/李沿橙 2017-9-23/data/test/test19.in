5
50
998 1 1000 1 998 999 1000 999 999 1000 998 998 1 1000 999 1 998 998 1000 1 999 999 998 1 998 1000 999 998 1000 1 1 1 999 1 998 998 998 1000 999 998 1 998 999 999 1 1000 1000 1000 999 1000 
28
11 985 4 11 4 2 8 7 1 11 983 10 9 5 992 11 6 9 9 1 5 994 988 989 7 3 4 995 
47
995 997 13 11 13 993 13 4 995 995 4 994 1000 10 997 998 994 5 997 995 1000 997 11 10 996 14 998 1000 993 14 14 997 994 1000 9 994 7 12 7 1000 1000 993 999 4 994 998 996 
50
397 56 305 341 50 213 383 200 9 246 228 413 175 237 62 160 70 421 388 431 321 314 141 400 311 336 61 418 161 358 316 406 213 259 28 130 406 121 166 374 408 229 372 79 323 22 198 16 134 178 
43
597 602 588 581 588 565 598 576 600 575 566 595 602 586 578 588 603 565 593 597 565 579 603 583 564 609 592 579 593 592 581 580 618 587 616 611 568 566 577 572 599 619 569 
