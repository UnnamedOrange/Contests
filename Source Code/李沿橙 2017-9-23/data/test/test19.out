1 1 1 1 1 1 1 1 1 1 1 1 998 998 998 998 998 998 998 998 998 998 998 998 998 999 999 999 999 999 999 999 999 999 999 999 999 1000 1000 1000 1000 1000 1000 1000 1000 1000 1000 1000 1000 998 
1 1 2 3 4 4 4 5 5 6 7 7 8 9 9 9 10 11 11 11 983 988 989 992 994 995 985 11 
4 4 4 5 7 7 9 10 10 11 11 12 13 13 13 14 14 993 993 994 994 994 994 994 995 995 995 995 996 996 997 997 997 997 997 998 998 998 999 1000 1000 1000 1000 1000 1000 993 14 
9 16 22 28 50 56 61 70 79 130 134 141 160 161 166 175 198 200 213 213 228 229 246 305 311 314 321 323 336 341 358 372 383 388 397 400 406 406 408 413 418 421 431 374 316 259 237 178 121 62 
564 565 565 565 566 566 568 569 575 576 577 578 579 579 581 581 583 586 587 588 588 592 592 593 593 597 597 598 599 600 602 602 603 609 616 618 619 611 603 595 588 580 572 
